mesh/profiles/urls.py

```diff
@@ -1,5 +1,6 @@
-from django.urls import path
+from django.urls import path, include
 from . import views as profile_views
+from django.http import HttpRequest
```

Comment:
Small nitpick, but is importing inlcude and HttpRequest necessary for this file? I can see that include might be necessary in the future due to some other urls.py files importing it as well, but I don't see how HttpRequest is necessary.

Reply:
I think this was merged from another branch, but your reasoning makes sense, will remove